impl/proto/mutation_v1_service/mutation_v1_service.proto

```diff
@@ -0,0 +1,49 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+// Mutation Service
+//
+// The Key Transparency mutation server service consists of APIs to fetch
+// mutations.
+package mutation.v1.service;
+
+import "github.com/google/keytransparency/core/proto/keytransparency_v1_types/keytransparency_v1_types.proto";
+import "google/api/annotations.proto";
+
+// The Mutation Service API represents a directory of mutations:
+//
+// - Mutations can be collected using the GetMutation APIs. Mutation resources
+//   are named /v1/mutations/* and /v1/mutations/*/stream.
+//
+service MutationService {
+  // GetMutations returns a list of mutations in a specific epoch.
+  //
+  // Returns a list of mutations and their inclusion proofs along with the epoch
+  // signed map root.
+  rpc GetMutations(keytransparency.v1.types.GetMutationsRequest)
+      returns (keytransparency.v1.types.GetMutationsResponse) {
+    option (google.api.http) = { get: "/v1/epochs" };
+  }
+
+  // GetMutationsStream is a streaming API similar to GetMutations.
+  //
+  // Returns a list of mutations and their inclusion proofs along with the epoch
+  // signed map root.
+  rpc GetMutationsStream(keytransparency.v1.types.GetMutationsRequest)
+      returns (stream keytransparency.v1.types.GetMutationsResponse) {
+    option (google.api.http) = { get: "/v1/epochs/stream" };
```

Comment:
Take a look at the API naming guidelines for custom verbs
https://cloud.google.com/apis/design/custom_methods